### PR TITLE
Fix Return values of various functions

### PR DESCRIPTION
**Pull request by seamuslee001**
@eileenmcnaughton

### CODE REVIEW

File: CRM/Extendedreport/Form/Report/ExtendedReport.php [hunk 1]
```diff
@@ -6349,9 +6350,9 @@ protected function alterFinancialType($value, &$row, $selectedField, $criteriaFi
    * @param $value
    * @param $row
    *
-   * @return array
+   * @return array|string
    */
-  protected function alterContributionStatus($value, &$row): array {
+  protected function alterContributionStatus($value, &$row) {
```
Comment (seamuslee001) [inline CRM/Extendedreport/Form/Report/ExtendedReport.php:6355]: if there is a $value then we get a string back but if no value then we get an array

File: CRM/Extendedreport/Form/Report/ExtendedReport.php [hunk 2]
```diff
@@ -8952,10 +8955,9 @@ protected function getAggregateRowFieldTitle(): string {
   }
 
   /**
-   * @return array
    * @throws \CiviCRM_API3_Exception
    */
-  protected function wrangleColumnHeadersForContributionPivotWithReceiveDateAggregate(): array {
+  protected function wrangleColumnHeadersForContributionPivotWithReceiveDateAggregate(): void {
```
Comment (seamuslee001) [inline CRM/Extendedreport/Form/Report/ExtendedReport.php:8960]: This function doesn't seem to return anything

**Comment by eileenmcnaughton** [comment]
oh - I didn't realise you were doing this - I made it stale :-(

**Comment by eileenmcnaughton** [comment]
Hmm conflicted ..

**Pull request closed without merging**